### PR TITLE
fix(eventstream): Fix inserting transactions to the SnubaEventStream

### PR DESCRIPTION
**Pull request by lynnagara**
Transactions should not be posted to both "transactions" and "events".
This was a relic from the past when events and transactions shared
a database table but this hasn't been the case for over a year now.

This backend is only used for tests / dev.

### CODE REVIEW

File: src/sentry/eventstream/snuba.py [hunk 1]
```diff
@@ -336,24 +336,18 @@ def _send(
 
         data = (self.EVENT_PROTOCOL_VERSION, _type) + extra_data
 
-        # TODO remove this once the unified dataset is available.
-        # Inserting into both events and transactions datasets lets us
-        # simulate what is currently happening via kafka when both the events
-        # and transactions consumers are running.
-        datasets = ["events"]
+        dataset = "events"
```
Comment (onewland) [inline src/sentry/eventstream/snuba.py:339]: should this be `errors` eventually?

**Review by onewland — APPROVED** [review]
seems fine to me assuming you've tested e2e locally

File: src/sentry/eventstream/snuba.py (hunk 1 above)
Comment (lynnagara) [reply src/sentry/eventstream/snuba.py:339]: No, the name of the dataset is events and did not change.

Comment (lynnagara) [reply src/sentry/eventstream/snuba.py:339]: https://github.com/getsentry/snuba/blob/master/snuba/datasets/factory.py#L56

**Pull request merged**